### ISSUE TITLE
Custom Derivation paths broken

### ISSUE THREAD

**Issue by jcallin** [MyEtherWallet/MyEtherWallet#1083]
- [ ] Feature request
  - [ x] Bug report

- **Bug Report**
  - What is the current behavior?
Custom derivation paths for Ethereum do not work
  - If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem
Go to the MEW webpage,
Access My Wallet -> Software -> Mnemonic Phrase -> 24 word -> No password -> HD Derivation Path -> Add Custom Path -> m/44'/60'/0' "Eth ledger" (I am trying to recover a legacy Ledger wallet generated with the almost-BIP44 derivation path) -> Select the custom path I just added -> **NO ADDRESSES CHANGE AND THE DERIVATION PATH LABEL STILL SAYS "ETHEREUM (TREZOR)"**
The dropdown UI for my custom path appears to be bugged as well, with a white box off to the right inside the dropdown
<img width="526" alt="Screen Shot 2019-05-18 at 1 05 47 PM" src="https://user-images.githubusercontent.com/10102484/57974489-d137c500-796d-11e9-8e93-00ca42fd832d.png">
^ After I click the path, no addresses change and the label still says "ETH (Trezor)"
  
- What is the expected behavior?
Derivation path changes to my custom path deriving on m/44'/60'/0'
- What is the motivation / use case for changing the behavior?
Users are unable to use wallet derived on a custom path, or use the custom path feature at all
- Please tell us about your environment:
- Operating system
OSX Mojave 10.14.3
- Browser and version
Chrome Version 74.0.3729.131 (Official Build) (64-bit)
Firefox 66.0.2 (64-bit)

**Comment by jcallin** [comment]
Correcting the above: the path I'm trying to derive on is the old Ledger path m/44'/60'/0' (unused `change` address)
It appears the 'new' Ledger path since Ledger Live is m/44'/60'/0'/0 (with the `change` address)

**Comment by jcallin** [comment]
Update
It looks like if your custom derivation path is the same as the Eth (Trezor) one, the UI stays on Eth (Trezor). I added the `m/44'/60'/0'` path as a custom path and it showed no wallet found. When I pressed "Access my wallet" I got this exception which I assume is because the derivation path without the `change` address cannot be read.
```
{"exception":{"values":[{"type":"TypeError","value":"Cannot read property 'getAddressString' of null","stacktrace":{"frames":[{"colno":303387,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"HTMLButtonElement.r","in_app":true,"lineno":448},{"colno":51712,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"HTMLButtonElement.Ja.i._wrapper","in_app":true,"lineno":42},{"colno":13533,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"HTMLButtonElement.n","in_app":true,"lineno":42},{"colno":11644,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"nt","in_app":true,"lineno":42},{"colno":428343,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/chunk-2a57289d.de77c1cd.js","function":"click","in_app":true,"lineno":28},{"colno":433941,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/chunk-2a57289d.de77c1cd.js","function":"o.unlockWallet","in_app":true,"lineno":28},{"colno":3205,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"h.dispatch","in_app":true,"lineno":58},{"colno":6843,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"h.dispatch","in_app":true,"lineno":58},{"colno":5532,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"Array.<anonymous>","in_app":true,"lineno":58},{"colno":235856,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/app.6e10c308.js","function":"h.Lt","in_app":true,"lineno":1},{"colno":3255,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"commit","in_app":true,"lineno":58},{"colno":6414,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"h.commit","in_app":true,"lineno":58},{"colno":7962,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"h._withCommit","in_app":true,"lineno":58},{"colno":6439,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"?","in_app":true,"lineno":58},{"colno":null,"filename":"<anonymous>","function":"Array.forEach","in_app":true,"lineno":null},{"colno":6459,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"?","in_app":true,"lineno":58},{"colno":5423,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/vendors.dee28920.js","function":"?","in_app":true,"lineno":58},{"colno":208980,"filename":"file:///MyEtherWallet-v5.0.11-ens.1/js/app.6e10c308.js","function":"h.X","in_app":true,"lineno":1}]}}]},"event_id":"1990316ea0504b40a8fb7e4dcf8a1d3d","platform":"javascript","sdk":{"name":"sentry.javascript.browser","packages":[{"name":"npm:@sentry/browser","version":"5.1.2"}],"version":"5.1.2","integrations":["InboundFilters","FunctionToString","TryCatch","Breadcrumbs","GlobalHandlers","LinkedErrors","UserAgent","Vue","vue"]},"environment":"web","release":"5.0.11-ens.1","extra":{"componentName":"component <network-and-address-modal>","propsData":{"hardwareWallet":{"identifier":"mnemonic","isHardware":false,"needPassword":true,"mnemonic":"body night cheap tackle bird blast update useful gate grass weather myself grocery divert measure rice neck govern enjoy mesh festival reform biology shift","password":"","supportedPaths":[{"path":"m/44'/60'/0'/0","label":"Ethereum (Trezor)"},{"path":"m/44'/61'/0'/0","label":"Ethereum Classic"},{"path":"m/44'/1'/0'/0","label":"Ethereum Testnet (Ropsten)"},{"path":"m/0'/0'/0'","label":"SingularDTV"},{"path":"m/44'/40'/0'/0","label":"Expanse"},{"path":"m/44'/108'/0'/0","label":"Ubiq"},{"path":"m/44'/163'/0'/0","label":"Ellaism"},{"path":"m/44'/1987'/0'/0","label":"EtherGem"},{"path":"m/44'/820'/0'/0","label":"Callisto"},{"path":"m/44'/1128'/0'/0","label":"Ethereum Social"},{"path":"m/44'/184'/0'/0","label":"Musicoin"},{"path":"m/44'/6060'/0'/0","label":"GoChain"},{"path":"m/44'/2018'/0'/0","label":"EOS Classic"},{"path":"m/44'/200625'/0'/0","label":"Akroma"},{"path":"m/44'/31102'/0'/0","label":"EtherSocial Network"},{"path":"m/44'/164'/0'/0","label":"PIRL"},{"path":"m/44'/1313114'/0'/0","label":"Ether-1"},{"path":"m/44'/1620'/0'/0","label":"Atheios"},{"path":"m/44'/889'/0'/0","label":"TomoChain"},{"path":"m/44'/76'/0'/0","label":"Mix Blockchain"},{"path":"m/44'/1171337'/0'/0","label":"Iolite"},{"path":"m/44'/137'/0'/0","label":"RSK Mainnet"},{"path":"m/44'/37310'/0'/0","label":"RSK Testnet"},{"path":"m/44'/1001'/0'/0","label":"ThunderCore"},{"path":"m/44'/997'/0'/0","label":"Solidum"},{"path":"m/44'/916'/0'/0","label":"Metadium"},{"path":"m/44'/237'/0'/0","label":"DEXON Network"}],"basePath":"m/44'/60'/0'/'","hdKey":{"xpriv":"my-private-key","xpub":"my-public-key"}}},"lifecycleHook":"v-on handler"},"request":{"url":"file:///MyEtherWallet-v5.0.11-ens.1/index.html#/access-my-wallet","headers":{"User-Agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36"}},"tags":{"network":"ETH","service":"myetherwallet.com","walletType":""}}
```

**Comment by jcallin** [comment]
Thanks everyone who worked on this. I entered the derivation path `m/44'/60'/0'` and was able to recover my old wallets! I've confirmed it is fixed with 5.0.13
![image](https://user-images.githubusercontent.com/10102484/59057324-ffaf0e80-884e-11e9-985a-ce871a9d607d.png)

**Comment by jcallin** [comment]
Thanks to everyone who worked on this. I confirmed I could view the wallets on 5.0.13 with derivation path `m/44'/60'/0'/'`

The first time I pressed "Access My Wallet" I got another strange error message, but I went back and tried a second time and it worked okay and I'm able to fully access the wallets. Might be worth investigating because it looked like the same message in t#1137 which is marked as closed, sorry I didn't grab the error message when it first came up.
